content/kubermatic/master/guides/monitoring_logging_alerting/user_cluster/admin_guide/_index.en.md

```diff
@@ -251,6 +273,18 @@ data:
 
 After the new dashboards are applied to the Seed Cluster, they will become available across all Grafana Organizations, and they can be found in the Grafana UI under Dashboards -> Manage.
 
+### Managing Alerting and Recording Rules
+
+Similar to managing Grafana Dashboards, KKP administrators can also define Prometheus-compatible rules for metrics and logs and make them available across all KKP user clusters with MLA enabled.
```

Comment:
I would emphasize the automation a bit: 
 - add "pre-" to "define": "administrators can also pre-define"
 - add "automatically" before "available": "make them automatically available across"